aws-codeguruprofiler-profilinggroup/src/test/java/software/amazon/codeguruprofiler/profilinggroup/UpdateHandlerTest.java

```diff
@@ -58,23 +69,221 @@
     private final String revisionId = "TestRevisionId-123";
 
     private final GetPolicyRequest getPolicyRequest =
-        GetPolicyRequest.builder().profilingGroupName(profilingGroupName).build();
+            GetPolicyRequest.builder().profilingGroupName(profilingGroupName).build();
+
+    private final GetNotificationConfigurationRequest getNotificationConfigurationRequest =
+            GetNotificationConfigurationRequest.builder().profilingGroupName(profilingGroupName).build();
+
+    private final RemoveNotificationChannelRequest removeNotificationChannelRequest =
+            RemoveNotificationChannelRequest.builder()
+                    .profilingGroupName(profilingGroupName)
+                    .channelId("channelId")
+                    .build();
+
+    private final AddNotificationChannelsRequest addNotificationChannelsRequest = AddNotificationChannelsRequest.builder()
+            .profilingGroupName(profilingGroupName)
+            .channels(Channel.builder()
+                    .uri("channelUri2")
+                    .eventPublishers(ImmutableSet.of(EventPublisher.ANOMALY_DETECTION))
+                    .build())
+            .build();
```

Comment:
Since this is only used in the `@Nested class WhenNotificationChannelConfigurationIsProvided`, can you move it inside that class?